src/pages/nft/class/_classId/index.vue

```diff
@@ -404,10 +416,15 @@ export default {
       isCollecting: false,
 
       isGiftDialogOpen: false,
+      isTippingDialogOpen: false,
+      isTriggerFromEditionSelector: false,
       giftSelectedValue: 0,
       shouldShowCollectionItem: false,
 
       trimmedCount: 10,
+
+      customPrice: -1,
```

Comment:
I think `0` is a better value for `customPrice`, since we don't really care if user explicitly set it to 0 or it just defaulted to 0